File: Planner/src/app/acceuil/info-sup.ts
export class InfoSup {
    public idInfo: number;
    public typeAvion: string;
    public transporteur: string;
    public pisteDecolage: string;
    public destination: string;
    public  altitude: string;
    public  vitesse: Number;
    public orientation: string;
    public  distance: Number;


    constructor(typeAvion: string, transporteur: string, pisteDecolage: string,
        destination: string, altitude: string, vitesse: Number, orientation: string, distance: Number) {
        this.typeAvion = typeAvion;
        this.transporteur = transporteur;
        this.pisteDecolage = pisteDecolage;
        this.destination = destination;
        this.altitude = altitude;
        this.vitesse = vitesse;
        this.orientation = orientation;
        this.distance = distance;
        }
}

File: API/API/Models/InfoSup.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace API.Models
{
    public class InfoSup
    {

           [Key]
        public int idInfo { get; set; }

      
        public string typeAvion { get; set; }

    
        public string transporteur    { get; set; }

      
        public string pisteDecolage { get; set; }
        
        public string destination { get; set; }

      
        public string altitude { get; set; }


      
        public int vitesse { get; set; }

       
        public string orientation { get; set; }

       
        public int distance { get; set; }


        public ICollection<Vol> Vol { get; set; }
    }
}